test/utils/dependency-initialiserSpec.js

```diff
@@ -4,7 +4,7 @@ var DependencyInitialiser = require( '../../src/utils/dependency-initialiser' ),
 	LoggerMock = require( '../mocks/logger-mock' ),
 	EventEmitter = require( 'events' ).EventEmitter;
 
-describe( 'dependency-initialiser', function(){
+xdescribe( 'dependency-initialiser', function(){
```

Comment:
Great find, really helpful. I think what happens is that one of the Plugin Mocks doesn't trigger a ready event. Whilst the test itself doesn't do too much, it does instantiate a dependency initialiser - so would very much like to leave it in and fix it, rather that #technicaldebt